src/components/modules/ModuleCheckout.tsx

```diff
@@ -84,7 +84,10 @@ export const ModuleCheckout = (props: ModuleCheckoutProps) => {
   );
 
   return ctaText ? (
-    <PressableModuleBase onPress={onPress}>
+    <PressableModuleBase
+      onPress={onPress}
+      accessibilityLabel={`${title} , ${ctaText}`}
```

Comment:
What if I need to listen to the details of the payment method, such as the value of the email address associated with it? Looking at the available examples, we can set the `subtitle` optional field, but it's completely ignored with this template string configuration. What do you think?